### ISSUE TITLE
nodifications don't work after upgrade from Mint 20.2 Cinnamon to Mint 20.3 Cinnamon

### ISSUE THREAD

**Issue by Foexle11** [linuxmint/cinnamon#10592]
```
 * Cinnamon version 5.2.7
   "Please specify if you are using the daily builds PPA"
   - I guess not, if thats not default ^^
 * Distribution - Linux Mint 20.3 Cinnamon
 * Nvidia RTX 2070 Super (nvidia-driver-495.46)
 * 64 Bit
 * 
 [syslog.txt](https://github.com/linuxmint/cinnamon/files/7836349/syslog.txt)
 [xsession-errors.txt](https://github.com/linuxmint/cinnamon/files/7836347/xsession-errors.txt)
 ```

**Issue**
Nodifications don't show up after Update to Mint 20.3 Cinnamon from Mint 20.2 Cinnamon. Test Message in "Meldungen" (don't know how its translated in english exactly, should be nodifications or messages or something) crashes the window and don't show anything. Also Tried with Firefox nodifications, Signal nodifications and Discord nodifications (crashes by any nodification I get, might me an related issue).

When starting Steam for the first time i need to log in into Mint again, might be related, might be something else, but I'll just tell you cause it could perhaps help?
X-App-Status-Applet seems to be involved aswell, doesn't show Steam or Discord unless removed and readded. (xapp-status@cinnamon.org)

**Steps to reproduce**

- Get any nodification from a program or hit the Test nodification in the settings.

**Expected behaviour**

Show a nodification Bubble in the top right corner, make a sound.

**Other information**
- I'm not shure if it's an bug or me being a noobie to Linux Mint. Any Help appreciated. If I can provide any more files or info, feel free to ask.

**Configuration info, if applicable**
If this bug report is related to an Applet, Desklet or Extension, please paste (or use a pastebin service) the offending extension's settings, if it has any.  You can obtain this by opening its configuration, clicking the 'hamburger' button in the upper-right corner of the window, and selecting "Export to a file". **Please be sure to review the contents and remove any personal data it may contain.**

- Not shure if 100% related or not

**Comment by Secret-chest** [comment]
It works for me.

**Comment by Foexle11** [comment]
Okay,
I dont know what I did wrong last time.
I just tried reinstalling the update cause "it might just be a corrupt file or something, why not just try it" by loarding a backup of the 20.2 and redo the installing process. And now it seems to work. Test nodification went trough. Might just been an updater hickup the other day. Even the Steam and Discord Crashes are gone. I'll still will keep an eye on it but it seems like this case is closed.
Thanks for your time and sorry I guess?
